EssentialsSpawn/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
+    implementation project(':EssentialsX')
+}
+
+shadowJar {
+    dependencies {
+        include (dependency('io.papermc:paperlib'))
+    }
+    relocate 'io.papermc.lib', 'com.earth2me.essentials.spawn.paperlib'
```

Comment:
We could consider using EssentialsX's shaded PaperLib instead of shading our own